spec/system/authorizations_spec.rb

```diff
@@ -74,8 +74,6 @@ def fill_in_authorization_form
 
         within ".authorizations-list" do
           expect(page).to have_content("El padró")
-          expect(page).not_to have_link("El padró")
-          expect(page).to have_content(I18n.localize(authorization.granted_at, format: :long, locale: :ca))
```

Comment:
Same as in the other PR, why is this removed? 😕

Reply:
They were both failing and I did a little digging to see it was removed in this PR in decidim, https://github.com/decidim/decidim/pull/2311/files

Reply:
But that PR is from like 3 years ago, and released under Decidim v0.9, so there must be something else going on... 😕